wallet/src/de/schildbach/wallet/ui/dashpay/ContactsActivity.kt

```diff
@@ -220,4 +238,8 @@ class ContactsActivity : GlobalFooterActivity(), TextWatcher, ContactSearchResul
         }
         return super.onOptionsItemSelected(item)
     }
+
+    override fun onViewAllRequests() {
+        startActivity(ContactsActivity.createIntent(this, ContactsActivity.MODE_VIEW_REQUESTS))
```

Comment:
`ContactsActivity` can be removed from this line in the `.createIntent` call and the `.MODE_REVIEW_REQUESTS` access.

Reply:
yes, and its removed